app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -67,6 +76,18 @@ class AdvancedFilter extends React.Component {
     });
   }
 
+  componentDidUpdate(prevProps) {
+    // Get list of assigned users for assigned users multi-select advanced filter
+    if (this.props.assigned_users !== prevProps.assigned_users) {
+      let index = advancedFilterOptions.findIndex(x => x.name === 'assigned-user');
+      let assigned_users = _.values(this.props.assigned_users).map(assigned_user => {
+        return { value: assigned_user, label: assigned_user };
+      });
+
+      advancedFilterOptions[Number(index)].options = assigned_users;
```

Comment:
so it seems like the options for assigned users here is based on the current jurisdiction of patients displayed on the dashboard which I think is fine when we're not filtering by a different jurisdiction or jurisdictions, in which case I think we would need to update these options to reflect the assigned users of the different jurisdiction or jurisdictions. let me know if that doesn't make sense and/or if you need help making that work

Reply:
do you mean filtering by jurisdictions in the advanced filter or in the input that is located at the top of the patients table?

Reply:
Are you suggesting to always include all assigned users here?

Reply:
This is what I asked you/the PO team on Slack about. Becky said for all available Assigned Users in the monitoree's jurisdiction be available to select regardless of whether or not a Jurisdiction filter has been added.

Reply:
alright, in that case we will need to get the list of `assigned_users` of the current user's jurisdiction. currently, this list is populated based on the assigned users from the jurisdiction being filtered (not the new jur adv filter)